### DISCUSSION TITLE
Add labels to dependency dashboard

### DISCUSSION THREAD

**Discussion by BryanHunt** [renovatebot/renovate#11341] [Request Help]
### What would you like Renovate to be able to do?

It would be nice to have a configuration option to add one or more labels to the dependency dashboard issue.  My primary use-case is filtering out the dashboards when looking at issues across multiple projects in a GitLab group.

### If you have any ideas on how this should be implemented, please tell us here.

None

### Is this a feature you are interested in implementing yourself?

No

**Comment by viceice** [comment]
👉 `dependencyDashboardLabels`

https://docs.renovatebot.com/configuration-options/#dependencydashboardlabels

**Comment by BryanHunt** [comment]
Doh!  I missed that.  Thanks!
